### PR TITLE
Feature/implement blocks page

### PR DESCRIPTION
**Pull request by VargaElod23**

### CODE REVIEW

**Comment by VargaElod23** [comment]
image

File: services/explorer/src/components/TransactionRow/TransactionRow.tsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
 import { Icons, Label } from '@taraxa_project/taraxa-ui';
 import moment from 'moment';
 import React from 'react';
+import { BlockData } from '../../pages/Blocks/Blocks.effects';
```
Comment (Malak67) [inline services/explorer/src/components/TransactionRow/TransactionRow.tsx:4]: Ideally we should not import files from effects in components. If we use the files here, then we should have the BlockData somewhere outside, in a models folder. Same goes for the imports from Transaction.effects. Let's move these models outside (in a models folder, table.ts file) and then import them here and in the effects as well.

**Review by Malak67 — APPROVED** [review]
LGTM. One minor comment to move the models outside.

File: services/explorer/src/components/TransactionRow/TransactionRow.tsx [hunk 2]
```diff
@@ -62,3 +63,41 @@ export const toTableRow = (props: TransactionData) => {
     ],
   };
 };
+
+export const toBlockTableRow = (props: BlockData) => {
+  const { timestamp, block, txHash, transactionCount } = props;
+  let age = Math.floor(+new Date() / 1000 - +timestamp);
```
Comment (Malak67) [inline services/explorer/src/components/TransactionRow/TransactionRow.tsx:69]: Can we move this in an utils function outside?

**Pull request merged**